test/extended/builds/forcepull.go

```diff
@@ -152,31 +100,31 @@ var _ = g.Describe("[LocalNode][builds] forcePull should affect pulling builder
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
 
-		g.It("ForcePull test case execution", func() {
-
-			g.By("when s2i force pull is false")
-
-			doTest(buildPrefixFS, "s2i false app/lang build", true, oc)
+		g.It("ForcePull test case execution s2i", func() {
 
 			g.By("when s2i force pull is true")
+			// run twice in case builder image was created on different node
+			// from the node processing this build
+			scrapeLogs(buildPrefixTS, oc)
+			scrapeLogs(buildPrefixTS, oc)
 
-			doTest(buildPrefixTS, "s2i true app/lang build", false, oc)
-
-			g.By("when docker force pull is false")
+		})
 
-			doTest(buildPrefixFD, "dock false app/lang build", true, oc)
+		g.It("ForcePull test case execution docker", func() {
 
 			g.By("docker when force pull is true")
+			// run twice in case builder image was created on different node
+			// from the node processing this build
```

Comment:
sure thing ... will let the in progress extended test run complete before updating commit